packages/client/src/__tests__/tlog/verify/checkpoint.test.ts

```diff
@@ -43,55 +43,34 @@ describe('verifyCheckpoint', () => {
   const checkpoint =
     'rekor.sigstore.dev - 2605736670972794746\n21428036\nrxnoKyFZlJ7/R6bMh/d3lcqwKqAy5CL1LcNBJP17kgQ=\nTimestamp: 1688058656037355364\n\n— rekor.sigstore.dev wNI9ajBFAiEAuDk7uu5Ae8Own/MjhSZNuVzbLuYH2jBMxbSA0WaNDNACIDV4reKpYiOpkwtvazCClnpUuduF2o/th2xR3gRZAUU4\n';
 
-  const inclusionProof: TransparencyLogEntry['inclusionProof'] = fromPartial({
-    checkpoint: { envelope: checkpoint },
-    rootHash: Buffer.from(
-      'rxnoKyFZlJ7/R6bMh/d3lcqwKqAy5CL1LcNBJP17kgQ=',
-      'base64'
-    ),
-  });
+  const inclusionProof: TLogEntryWithInclusionProof['inclusionProof'] =
+    fromPartial({
+      checkpoint: { envelope: checkpoint },
+      rootHash: Buffer.from(
+        'rxnoKyFZlJ7/R6bMh/d3lcqwKqAy5CL1LcNBJP17kgQ=',
+        'base64'
+      ),
+    });
 
-  const entry: TransparencyLogEntry = fromPartial({
+  const entry: TLogEntryWithInclusionProof = fromPartial({
     inclusionProof: inclusionProof,
     integratedTime: '1688058655',
   });
 
   describe('when the entry has a valid checkpoint', () => {
-    it('does NOT throw an error', () => {
-      expect(() => verifyCheckpoint(entry, tlogs)).not.toThrow();
-    });
-  });
-
-  describe('when the entry has no inclusion proof', () => {
-    const entryWithoutProof: TransparencyLogEntry = fromPartial({
-      logId: { keyId: keyID },
-    });
-
-    it('throws a VerificationError', () => {
-      expect(() => verifyCheckpoint(entryWithoutProof, tlogs)).toThrow(
-        VerificationError
-      );
-    });
-  });
-
-  describe('when the entry has no checkpoint', () => {
-    const entryWithoutCheckpoint: TransparencyLogEntry = fromPartial({
-      inclusionProof: {},
-    });
-
-    it('throws a VerificationError', () => {
-      expect(() => verifyCheckpoint(entryWithoutCheckpoint, tlogs)).toThrow(
-        VerificationError
-      );
+    it('returns true', () => {
+      expect(verifyCheckpoint(entry, tlogs)).toBe(true);
```

Comment:
Since we're using the narrowed `TLogEntryWithInclusionProof` here we no longer need to perform these checks for undefined fields (this happens as part of the bundle validation step)